examples/rollup/README.md

```diff
@@ -21,8 +21,13 @@ pip3 install -r requirements.txt
 
 ```
 Run 
-yarn install --> installs all packages
-yarn start --> runs rollup to compile and starts the server
+yarn install # installs all packages
```

Comment:
Did you also mean to edit the package.json similar to https://github.com/stripe/connect-js/pull/55/files#diff-9722ed0a23a99f56ea234e553da55c4adddbc8c2c3649d4299ecc0643def7648?